Dockerfile

```
@@ -21,8 +21,9 @@ EXPOSE 4000
 CMD ["poetry", "run", "flask","run","--host=0.0.0.0"]
 
 FROM base as production
-CMD ["poetry", "run","gunicorn", "todo_app.app:app", "-b", "0.0.0.0:5000"]
+# CMD ["poetry", "run","gunicorn", "todo_app.app:app", "-b", "0.0.0.0:80"]
 
+CMD poetry run gunicorn todo_app.app:app -b 0.0.0.0:$PORT
```

Note that this changes your behaviour slightly so that the app will error if the user doesn't specify a PORT environment variable; so your Readme instructions will now fail! Adding in a default value, e.g. with the below, would help by keeping your instructions consistent but still allowing Azure to override the PORT if/when it wants to
```suggestion
ENV PORT=5000
CMD poetry run gunicorn todo_app.app:app -b 0.0.0.0:$PORT
```